internal/cli/atlas/backup/snapshots/list.go

```diff
@@ -15,6 +15,7 @@
 package snapshots
 
 import (
+	"fmt"
```

Comment:
the file is not formatted. Lint is failing. Run `make lint` to validate against our linting rules.